bindings/guile/test/test-business-core.scm

```diff
@@ -10,7 +10,7 @@
   (test-runner-factory gnc:test-runner)
   (test-begin "test-business-core")
   (core-tests)
-  (test-end "test-business-core"))
+  (zero? (test-runner-fail-count (test-end "test-business-core"))))
```

Comment:
This is ugly. Abstract it to
```
(define-public (gnc:test-end test-name)
   (let ((runner (test-runner-current)))
        (test-end test-name)
        (zero? (test-runner-fail-count runner))))
```
Then you can just change all of `(test-end "foo")` to `(gnc:test-end "foo")`.